src/clr-angular/forms/combobox/options.ts

```diff
@@ -1,59 +1,140 @@
 /**
- * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { Component, ElementRef, Inject, Injector, OnDestroy, Optional } from '@angular/core';
-import { Subscription } from 'rxjs';
-
-import { AbstractPopover } from '../../popover/common/abstract-popover';
-import { Point } from '../../popover/common/popover';
+import {
+  Component,
+  ContentChildren,
+  ElementRef,
+  Inject,
+  Input,
+  Optional,
+  QueryList,
```

Comment:
Needs a View Basics test that ensures this class is always added.

Reply:
Comment misplaced. If you are talking about `.clr-combobox-options`, we have such test.
If not, please specify which class do you mean.

Reply:
Confirmed. Fixed.